tidb-in-kubernetes/troubleshoot: notes about "cluster-id mismatch" (#1586)
pingcap/docs: dev/tidb-in-kubernetes/troubleshoot.md

After checking the error messages in the log, you can refer to [Cannot start `tidb-server`](/dev/how-to/troubleshoot/cluster-setup.md#cannot-start-tidb-server), [Cannot start `tikv-server`](/dev/how-to/troubleshoot/cluster-setup.md#cannot-start-tikv-server), and [Cannot start `pd-server`](/dev/how-to/troubleshoot/cluster-setup.md#cannot-start-pd-server) for further troubleshooting.

> When "cluster id mismatch" message appears in the TiKV Pod log, it means that the TiKV Pod may use old data from other or previous TiKV Pod. If the data on the local disk remain uncleared when configuring local storage in the cluster, or the data is not recycled by local volume provisioner program due to a forced deletion of PV, an error may occur.

Reviewer: - Please add an article for the singular form of a countable noun.
- Use `might` instead of `may` because `may` has multiple meanings.

```suggestion
When the "cluster id mismatch" message appears in the TiKV Pod log, it means that the TiKV Pod might have used old data from other or previous TiKV Pod. If the data on the local disk remain uncleared when you configure local storage in the cluster, or the data is not recycled by the local volume provisioner due to a forced deletion of PV, an error might occur.
```

Author: Got it.

> When you make sure that the TiKV should join the cluster as a new node and that the data on the PV should be deleted, you can delete the TiKV Pod and the corresponding PVC. The TiKV Pod will automatically rebuild and bind the new PV for use. When configuring local storage, local storage on the machine should be deleted to avoid Kubernetes using old data. As for operation and maintenance, the PV should be managed by the local volume provisioner program instead of being forcibly deleted. Users can manage the lifecycle of PV by creating, deleting PVCs and setting the reclaimPolicy for the PV.

Reviewer: **Please pay attention to the consistency of subjects. For example, `local storage` cannot configure `local storage`.**

```suggestion
If you are confirmed that TiKV should join the cluster as a new node and that the data on the PV should be deleted, you can delete the TiKV Pod and the corresponding PVC. The TiKV Pod automatically rebuilds and binds the new PV for use. When configuring local storage, delete local storage on the machine to avoid Kubernetes using old data. In cluster operation and maintenance, manage PV using the local volume provisioner and do not delete it forcibly. You can manage the lifecycle of PV by creating, deleting PVCs, and setting `reclaimPolicy` for the PV.
```
